src/Todo.tsx

```diff
@@ -1,27 +1,19 @@
 import React from "react";
 
-class Todo extends React.Component<any> {
-	shouldComponentUpdate(prevProps: any) {
-	if(this.props != prevProps) {
-		return true;
-	}
-		return false;
-	}
-
-	handleOnClick() {
-		window.location.href = '/detail'
-	}
+interface TodoProps {
```

Comment:
Provide which data you expect as the input for the component. Don't use <any>...